README.md

```diff
@@ -90,36 +90,53 @@ If you wish to use cmake to build Halide, the build procedure is:
 
     % mkdir cmake_build
     % cd cmake_build
-    % export LLVM_ROOT=/path/to/llvm3.7/build
-    % cmake -DLLVM_BIN=${LLVM_ROOT}/bin -DLLVM_INCLUDE="${LLVM_ROOT}/../include;${LLVM_ROOT}/include" -DLLVM_LIB=${LLVM_ROOT}/lib -DLLVM_VERSION=37 ..
+    % cmake -DLLVM_DIR=/path-to-llvm-build/lib/cmake/llvm -DCMAKE_BUILD_TYPE=Release -DLLVM_VERSION=37 /path/to/halide
     % make -j8
 
-#### Building Halide and llvm as 32-bit on 64-bit linux
+LLVM_DIR should be the folder in the LLVM installation or build tree that contains LLVMConfig.cmake.
 
-This is necessary if you want to JIT compile 32-bit code. It is not
-necessary for AOT compiling 32-bit Halide pipelines. The 64-bit
-version of Halide cross-compiles 32-bit code just fine.
+#### Building Halide and LLVM on Windows
 
-To get a 32-bit llvm, configure and compile it like so:
+These instructions assume you have MSVC 2015. MSBuild and cmake should
```

Comment:
Probably worth adding that we test with MSVC2015 Update 3; earlier Updates may have problems, so check this first.